components/src/core/InfoListItem/InfoListItem.tsx

```diff
@@ -211,6 +211,16 @@ export const InfoListItem: React.FC<InfoListItemProps> = (props) => {
                 }}
             />
             {getRightComponent()}
+        </>
+    );
+
+    return onClick ? (
+        <ListItem style={getWrapperStyle()} onClick={(): void => onClick()} dense={dense} button>
+            {listItemChild}
+        </ListItem>
+    ) : (
+        <ListItem style={getWrapperStyle()} dense={dense}>
+            {listItemChild}
         </ListItem>
```

Comment:
why is this change on the ListItem necessary?

Reply:
To add the Ripple. 

If onClick, `<ListItem button />`
else `<ListItem />` 

I wasn't sure of a better way to do it